### PR TITLE
Add users screen

### PR DESCRIPTION
**Pull request by jalalkhademi**
## Description
- Description of issue
- Output before solved
- Changes in Issue
- Output of solved issue

## Type of Changes?
Just put an x in the [ ] which are valid.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing Suites

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes.
- [ ] ./gradlew assembleDebug assembleRelease

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Comment by jalalkhademi** [comment]
New screen for latest chats and users is added. users UI is implemented as rows of main screen.

**Comment by iadeelzafar** [comment]
@jalalkhademi please add UI screenshot to the PR's description.

File: app/src/main/res/layout/activity_users.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerView"
+        android:layout_width="409dp"
```
Comment (iadeelzafar) [inline app/src/main/res/layout/activity_users.xml]: This shouldn't be fixed.

File: app/src/main/res/layout/activity_users.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        android:layout_height="729dp"
```
Comment (iadeelzafar) [inline app/src/main/res/layout/activity_users.xml]: This shouldn't be fixed.

File: app/src/main/res/layout/user_row.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/img_user"
+        android:layout_width="50dp"
```
Comment (iadeelzafar) [inline app/src/main/res/layout/user_row.xml]: This shouldn't be fixed.

File: app/src/main/res/layout/user_row.xml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        android:layout_height="50dp"
```
Comment (iadeelzafar) [inline app/src/main/res/layout/user_row.xml]: This shouldn't be fixed.

**Comment by jalalkhademi** [comment]
Dear iadeel,

here it is the screen shot of implemented screen. 

UsersScreen

**Pull request closed without merging**